### PR TITLE
Update location of the repo in the Readme

### PR DESCRIPTION
**Pull request by mlucas**

### CODE REVIEW

**Comment by mlucas** [comment]
This change is dumb.

**Comment by iamef** [comment]
Don't we need to be gracious professionals? Why are we saying this change is dumb?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Playground
 A playground where you can learn and experiment with git.
+
+# Location
```
Comment (mlucas) [inline README.md]: Let's call this Repository

**Review by mvoodarla — APPROVED** [review]
Solid change

**Review by tnarayanan — APPROVED** [review]
Looks good!

**Review by shruthij28 — COMMENTED** [review]
Looks good!

**Pull request merged**